src/clj/game/cards/assets.clj

```diff
@@ -585,7 +584,6 @@
    "Drudge Work"
    {:effect (effect (add-counter card :power 3))
     :abilities [{:cost [:click 1]
-                 :counter-cost [:power 1]
```

Comment:
I think, this can stay. Imho, the ruling would be that Drudge Work is already in the Archives, when it is doing the reshuffling of the last agenda..

You first pay the costs of the paid ability. Then there is a checkpoint, in which Drudge Work gets trashed because there are no power counters left. And then you're doing the ability, which exists even if the card got trashed.

Maybe we should ask the rules team on this.

Reply:
No, you’re totally right! I got over-zealous and forgot what the heck I was doing haha. This is the perfect place for `:counter-cost`.